src/test/ui/on-unimplemented/multiple-impls.stderr

```diff
@@ -7,14 +7,6 @@ error[E0277]: the trait bound `[i32]: Index<u32>` is not satisfied
    = help: the trait `Index<u32>` is not implemented for `[i32]`
    = note: required by `Index::index`
 
```

Comment:
Why are we losing errors here?

Reply:
Good question. I also do not understand why there's duplicate errors here in the first place.

Reply:
This needs to be investigated.